packages/workers-shared/asset-server-worker/src/index.ts

```diff
@@ -60,6 +65,6 @@ export default {
 		);
 		const headers = getMergedHeaders(request.headers, additionalHeaders);
 
-		return new OkResponse(assetContent, { headers, encodeBody: "automatic" });
-	},
-};
+		return new OkResponse(assetContent, { headers });
```

Comment:
`encodeBody` is `"automatic"` by default, so don't think we need to specify it here.